book/website/index.md

```diff
@@ -1,75 +1,124 @@
 (welcome)=
 # Welcome
 
-*Welcome to The Turing Way handbook to reproducible, ethical and collaborative data science.*
+_Welcome to The Turing Way Handbook to Reproducible, Ethical, and Collaborative Data Science._
 
-_The Turing Way_ is an open science, open collaboration, and community-driven project.
-We involve and support a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
-Our goal is to provide all the information that researchers, data scientists, software engineers, policymakers, and other practitioners in academia, industry, government and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
+_The Turing Way_ is an open source book, a global collaboration and a supportive community.
+We involve and collaborate with a diverse community of contributors to make data science accessible, comprehensible and beneficial for everyone.
 
 ```{admonition} Top Tip
 :class: tip
-_The Turing Way_ is not meant to be read from start to finish.
-Start with a concept, tool or method that you need now, in your current work.
-Browse the different guides that make up the book, or use the search box to search for whatever you would like to learn about first.
+_The Turing Way_ is not meant to be read from start to finish. Start with what interests you!
+Whether it's a concept, a tool, or a method you need right now, jump in.
+Browse the different guides or use the search box —
+it's all about what you want to learn.
 ```
 
-All stakeholders from any level of experience or domain expertise are encouraged to use _The Turing Way_ to understand their roles and responsibility of reproducibility in data science and research.
-You can inspect our resources on [GitHub](https://github.com/the-turing-way/the-turing-way), contribute to the project as described in our [contribution guidelines](https://github.com/the-turing-way/the-turing-way/blob/main/CONTRIBUTING.md) and re-use all materials ([see the License](https://github.com/the-turing-way/the-turing-way/blob/main/LICENSE.md)).
+Our moonshot goal is to make reproducibility "too easy not to do"! 
+Whether you are a researcher, coder, project leader, community connector, funder, policymaker, or just curious about data, your unique role is crucial in this journey.
 
-In the {ref}`Foreword Section<fw>` of this book, we provide meta-level information about _The Turing Way_, including guidance on how to use and navigate this project, how to cite the book and opportunities to get involved.
+Let's make data science inclusive, together! 
+Visit our [start page](https://the-turing-way.start.page/) to get started. 🚀
```

Comment:
Directing people to the start page feels a little circular - the first link on the start page links back to this page. 
The start page is great for a links summary for people to get more involved in the project - maybe we should introduce it with that context?

Reply:
```suggestion
```
This shouldn't be here as we do have a paragraph below.